drivers/regex/mlx5/mlx5_regex.h

```diff
@@ -44,5 +46,6 @@ int mlx5_regex_info_get(struct rte_regexdev *dev,
 			struct rte_regexdev_info *info);
 int mlx5_regex_configure(struct rte_regexdev *dev,
 			 const struct rte_regexdev_config *cfg);
-
+int mlx5_regex_rules_db_import(struct rte_regexdev *dev,
+		     const struct rte_regexdev_config *cfg);
```

Comment:
the second line should start from the (.
this is true for all.

Reply:
I use 3 tabs and a space above to align to (.  Are spaces not allowed?  I'll run checkpatch on all of this today anyway, but my code is aligned to that rule for the most part, so must me something I'm missing!

Reply:
first maybe what is displayed is not the correct. 
spaces are allowed if they are after tab and less than a tab.